qualang_tools/config/server/qua1_openapi.json

```diff
@@ -0,0 +1,550 @@
+{
```

Comment:
this is qua from qm-qua 0.3.4.
it might change. does any change in the json requires change here?

if you have to save it here, please rename it to reflect the version supported.

Reply:
The version of the schema that is saved here is already in the file ( "info" > "version" ). We don't allow selection currently of different schema, so we can just keep updated this file, without changing things on two places (file name and its reference in the code). Is this satisfactory for now (just one major qua version)?
Once there are multiple major versions we can have multiple files, but then it would make sense to support in GUI selection of appropriate.

Reply:
Now we download latest schema if possible (cfd000af8f1cf5041bf5596689e83a795a9ced64 3517089f8633691dbd19d78c298e0e5d14095b3b 42f689b43b7d26705c7b7444649dc159f4daaf49) , if not just use local copy